R/process_lookup_homelessness.R

```diff
@@ -0,0 +1,114 @@
+# It would probably be simplest to create a lookup from the Homelessness data of one row per CHI +
+#   start dates of all applications. This can then be matched to episode and individual to easily create the hl1_in_fy flag(s)
+# (if someone is in the lookup -> 1, otherwise -> 0). For the episode file, it is slightly more complicated to create the other flags,
+# either matching on all the data by CHI and then doing some comparison of dates (as in SPSS),
+# or using some of the fancy new joins (dplyr 1.1.0 joins) and doing it that way.
+#
+#
+
+
+# data <- slfhelper::read_slf_episode("1718", c("anon_chi", "record_keydate1", "record_keydate2", "recid"))
+
+# year <- "1718"
+
+
```

Comment:
I think all this can be removed now, or possibly integrated into the function documentation